databricks-sdk-java/src/main/java/com/databricks/sdk/core/DatabricksException.java

```diff
@@ -8,4 +14,11 @@ public DatabricksException(String message) {
   public DatabricksException(String message, Throwable cause) {
     super(message, cause);
   }
+
+  public List<ErrorDetail> getErrorInfo() {
+    if (!(this instanceof DatabricksError)) {
+      return Collections.emptyList();
+    }
+    return ((DatabricksError) this).getDetailsByType(ERROR_INFO_TYPE);
+  }
```

Comment:
This accessor can live in `DatabricksError`. The caller should catch `DatabricksError` and work with it directly. This error being thrown means the Databricks backend actually responded.

Reply:
Moved